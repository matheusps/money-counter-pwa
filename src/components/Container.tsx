import React, { FC } from 'react'

const Container: FC = ({ children }) => {
  return <div>{children}</div>
}

export default Container
