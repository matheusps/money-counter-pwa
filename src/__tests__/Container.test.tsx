import React from 'react'
import { render } from 'react-testing-library'
import 'jest-styled-components'

import Container from '../components/Container'

describe('Container component', () => {
  const component = render(<Container>Testing</Container>)

  it('should render', () => {
    expect(component).toBeTruthy()
  })

  it('should match snapshot', () => {
    expect(component.asFragment()).toMatchSnapshot()
  })
})
