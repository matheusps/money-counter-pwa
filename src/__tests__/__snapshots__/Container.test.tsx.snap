// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container component should match snapshot 1`] = `
<DocumentFragment>
  .c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  height: 100vh;
  padding: 3rem;
}

<div
    class="c0"
  >
    Testing
  </div>
</DocumentFragment>
`;
